fzf.el

```diff
@@ -321,13 +476,40 @@ DIRECTORY, if non-nil, is prepended to the result of fzf."
 
 ;; Internal helper function
 (defun fzf--action-find-file-with-line (target)
-  (fzf--action-find-file target)
-  (let* ((parts (split-string target ":"))
-         (f (expand-file-name (nth 0 parts))))
-    (when (file-exists-p f)
-      (find-file f)
-      (goto-char (point-min))
-      (forward-line (-  (string-to-number (nth 1 parts)) 1)))))
+  ;; (fzf--action-find-file target)
```

Comment:
dead code

Reply:
correct,  I though I had removed it.  I'll check.